supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchJob.scala

```diff
@@ -79,83 +85,177 @@ final case class AwsBatchJob(jobDescriptor: BackendJobDescriptor, // WDL/CWL
   }
-  lazy val logsclient = {
+  lazy val cloudWatchLogsClient: CloudWatchLogsClient = {
     val builder = CloudWatchLogsClient.builder()
-    configRegion.foreach(builder.region)
-    builder.build
+    configureClient(builder, optAwsAuthMode, configRegion)
+  }
+
+  lazy val s3Client: S3Client = {
+    val builder = S3Client.builder()
+    configureClient(builder, optAwsAuthMode, configRegion)
   }
 
-  lazy val reconfiguredScript = {
-    // We'll use the MD5 of the dockerRc so the boundary is "random" but consistent
-    val boundary = MessageDigest.getInstance("MD5").digest(dockerRc.getBytes).map("%02x".format(_)).mkString
-
-    // NOTE: We are assuming the presence of a volume named "local-disk".
-    //       This requires a custom AMI with the volume defined. But, since
-    //       we need a custom AMI anyway for any real workflow, we just need
-    //       to make sure this requirement is documented.
-    //
-    //       This MIME format is technically no longer necessary as the
-    //       proxy docker container will manage the stdout/stderr/rc
-    //       stuff being copied correctly, but this can still be useful
-    //       later, so I'm leaving it here for potential future use.
-    script.concat(s"""
-    |echo "MIME-Version: 1.0
-    |Content-Type: multipart/alternative; boundary="$boundary"
-    |
-    |--$boundary
-    |Content-Type: text/plain
-    |Content-Disposition: attachment; filename="rc.txt"
-    |"
-    |cat $dockerRc
-    |echo "--$boundary
-    |Content-Type: text/plain
-    |Content-Disposition: attachment; filename="stdout.txt"
-    |"
-    |cat $dockerStdout
-    |echo "--$boundary
-    |Content-Type: text/plain
-    |Content-Disposition: attachment; filename="stderr.txt"
-    |"
-    |cat $dockerStderr
-    |echo "--$boundary--"
-    |exit 0
-    """).stripMargin
+  /**
+    * The goal of the reconfigured script is to do 3 things:
+    * 1. make a directory "working_dir" and replace all "/cromwell_root" with that
+    * 2. before the command line, insert the input copy command to stage input files
+    * 3. at the end of the script sync all content with the s3 bucket
+    */
+  lazy val reconfiguredScript: String = {
+    //this is the location of the aws cli mounted into the container by the ec2 launch template
+    val s3Cmd = "/usr/local/aws-cli/v2/current/bin/aws s3"
+    //internal to the container, therefore not mounted
+    val workDir = "/scratch"
+    //working in a mount will cause collisions in long running workers
+    val replaced = commandScript.replaceAllLiterally(AwsBatchWorkingDisk.MountPoint.pathAsString, workDir)
+    val insertionPoint = replaced.indexOf("\n", replaced.indexOf("#!")) +1 //just after the new line after the shebang!
+
+    //generate a series of s3 copy statements to copy any s3 files into the container
+    val inputCopyCommand = inputs.map {
+      case input: AwsBatchFileInput if input.s3key.startsWith("s3://") && input.s3key.endsWith(".tmp") =>
```

Comment:
Only if that input file contains working directory paths otherwise the sed pattern won't match and nothing will happen to them. If they do contain those paths then they would almost certainly want to be translated otherwise they would be pointing to a non-existent path